cfg/config.yaml

```diff
@@ -133,3 +133,6 @@ plugins:
 
   depthcam:
     topic-pcl: "~/depthcam-pcl/"
+
+  object-tracking:
+    use-object-tracking: false
```

Comment:
This config name is confusing because it does not describe what it actually does. What about `enable-public-object-pose-publisher`?

Also, there is no need to nest this in `object-tracking`.